api4/system.go

```diff
@@ -68,6 +69,8 @@ func (api *API) InitSystem() {
 	api.BaseRoutes.System.Handle("/notices/view", api.ApiSessionRequired(updateViewedProductNotices)).Methods("PUT")
 
 	api.BaseRoutes.System.Handle("/support_packet", api.ApiSessionRequired(generateSupportPacket)).Methods("GET")
+	api.BaseRoutes.ApiRoot.Handle("/first_admin_visit_marketplace", api.ApiHandler(setFirstAdminVisitMarketplaceStatus)).Methods("POST")
+	api.BaseRoutes.ApiRoot.Handle("/first_admin_visit_marketplace", api.ApiSessionRequired(getFirstAdminVisitMarketplaceStatus)).Methods("GET")
```

Comment:
Nit pick Usually `GET` routes are placed before `POST` routes.

Reply:
I've seen it done differently in diff places though (including our own `.../api4/plugin.go`)

Reply:
Good point. Then it's just me wishing things are more consistent :wink: